tracking/src/main/java/org/hps/recon/tracking/SensorSetup.java

```diff
@@ -29,6 +33,10 @@ public SensorSetup() { }
      */
     public void setFittedHitCollection(String fittedHitColName) { fittedHitColName_ = fittedHitColName; }
 
+    public void setSkipSensors(String[] sensorNames) {
+        this.sensorNames_ = new ArrayList<String>(Arrays.asList(sensorNames));
```

Comment:
Change this to Hash Map. 

On a separate note, I wonder if this method should check if `sensorNames` was already initializing before doing it again.  I can see a situation where this gets set twice on accident in a steering file and could lead to unexpected behavior. Maybe if `sensorNames` has already been initialized, throw an exception?

Reply:
I think the framework already checks you only set a specific variable in the steering file once and throws an error if it is there more than once.

Reply:
Given the behavior we would like from this code, not clear to me HashMap really buys us anything.